example/src/App.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react'
 
 import {
   PaymentMethod,
+  RevOps,
```

Comment:
import { ..., RevOps } from 'revops-js' feels funky. 

Hypothetically rethink of the naming here, make the developer experience connect to existing notions of elements. 

import RevOps from 'revops-js'



```
<RevOps.Form {... auth code...} >
    <RevOps.PaymentMethod />
</RevOps.Form>
```